### PR TITLE
add docstring examples for moments_hu and centroid

### PR DESCRIPTION
**Pull request by grlee77**
## Description

This PR corrects a statement of ">4" to be ">=4" in the `moments_hu` docstring and adds a basic usage example.

## Checklist

- [x] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)~

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: skimage/measure/_moments.py [hunk 1]
```diff
@@ -352,6 +360,14 @@ def centroid(image):
     -------
     center : tuple of float, length ``image.ndim``
         The centroid of the (nonzero) pixels in ``image``.
+
+    Examples
+    --------
+    image = np.zeros((20, 20), dtype=np.double)
```
Comment (soupault) [inline skimage/measure/_moments.py:366]: Missing `>>> ` here:
```
_________________ [doctest] skimage.measure._moments.centroid __________________
358 
359     Returns
360     -------
361     center : tuple of float, length ``image.ndim``
362         The centroid of the (nonzero) pixels in ``image``.
363 
364     Examples
365     --------
366     image = np.zeros((20, 20), dtype=np.double)
367     >>> image[13:17, 13:17] = 0.5
UNEXPECTED EXCEPTION: NameError("name 'image' is not defined")
Traceback (most recent call last):
  File "/opt/python/3.7.1/lib/python3.7/doctest.py", line 1329, in __run
    compileflags, 1), test.globs)
  File "<doctest skimage.measure._moments.centroid[0]>", line 1, in <module>
NameError: name 'image' is not defined
/home/travis/build/scikit-image/scikit-image/skimage/measure/_moments.py:367: UnexpectedException
```

Comment (soupault) [inline skimage/measure/_moments.py:366]: This line currently does not pass the CI. I believe it should be changed to:
```suggestion
    >>> image = np.zeros((20, 20), dtype=np.double)
```

**Comment by emmanuelle** [comment]
@grlee77 I took the liberty to merge @soupault 's suggestion to your branch, I hope you don't mind :-). Travis takes ages to execute so I wanted to be sure that the CI is ok.

**Comment by grlee77** [comment]
Thanks @emmanuelle and @soupault. remaining failures are just the known Cython-related ones from #4624

**Comment by emmanuelle** [comment]
Thank you @grlee77 , merging.

**Pull request merged**